README.md

```diff
@@ -10,7 +10,25 @@ Production ready.
 
 ## Installation
 
-The changelog-tool requires [corral](https://github.com/ponylang/corral) to be installed.
+### Ponyup
+
+You can use [`ponyup`](https://github.com/ponylang/ponyup#installing-ponyup), our toolchain multiplexer, to download both `release` and `nighly` builds, if you're using either Linux or macOS. Windows still requires building from source
+
+Linux only:
+
+```bash
+ponyup update changelog-tool release
+```
+
```

Comment:
This should be deleted.

Reply:
Dammit, I've made a mistake, it should be:

```
Linux only:
ponyup update changelog-tool release
```

```
Linux and macOS:
ponyup update changelog-tool nightly
```

Searching for the release packages on [cloudsmith](https://cloudsmith.io/~ponylang/repos/releases/packages/?q=changelog-tool), the latest release exists for Linux Release and nightlies for Linux and only nightlies for macOS as described in the original issue, still applies? Otherwise, I'll leave only the latter installing instruction.

Reply:
Why do you have Linux twice?

Reply:
There's still only Linux. There's been no release of changelog-tool since that time.

Reply:
Ok, I was under the impression that we could only install `release` in Linux, and `nightly` on both Linux and macOS.

Reply:
Thats correct but its confusing to give people two options.